Dockerfile

```diff
@@ -179,11 +180,5 @@ ENV PYTHONPATH=/tuna
 
 RUN python3 setup.py install
 
-# Install SolverAnalytics dependencies (this require root access, so,
-# the dependencies must be installed here, and can't be installed in Jenkinsfile)
-RUN git clone https://$FIN_TOKEN:x-oauth-basic@github.com/ROCmSoftwarePlatform/SolverAnalytics.git
-RUN pip3 install --default-timeout=100000 -r SolverAnalytics/requirements.txt
-RUN rm -rf SolverAnalytics
-
```

Comment:
This can be bad in the long run. If a change to Tuna breaks SolverAnalytics, it might go unnoticed

Reply:
I noticed that the SolverAnalytics repo is mainly used by the test in vars/utils.groovy:323 solversAnalyticsTest
And it is installed at this point anyway.

Reply:
@cderb Right, but the dependencies for SolverAnalytics are installed here. Installing the deps requires root access and that's why it's done in the DockerFile

Reply:
It appears that CI passes without installing the dependencies at this point, Including the solverAnalyticsTest stage. And the docker had to rebuild because I updated the rocm and miopen versions. So this appears to be working fine? Why wouldn't we have root inside the docker container when we are installing SolverAnalytics during the jenkins stage?

Reply:
SolverAnalytics requires numpy and pandas, and it appears MITuna itself now has numpy and pandas included as requirements. That's why the CI is passing. However, if at a later point, SolverAnalytics takes on a new requirement, or Tuna gets rid of numpy/pandas from its requirements.txt, Tuna's CI will start erroring out.

Reply:
Let's add the install reqs at the same point in utils.groovy.